### DISCUSSION TITLE
New to appium and seeing a syntax error when trying to run appium in the command prompt. Please Help!

### DISCUSSION THREAD

**Discussion by zshig235** [appium/appium#18818] [Q&A]
I tried uninstalling and reinstalling appium but still have this issue where when i try to run appium the latest version, it throws the syntax error before appium starts. 
Here is a snippet of the error, you can view the image for more details.
Fatal SyntaxError: Expected ',' or '}' after property value in JSON at position 9558

Whats weird is that i installed the older version and it is able to start but i cant use that. 
The latest version was working for me last couple of weeks but started seeing this error today. 

<img width="661" alt="appium issue" src="https://github.com/appium/appium/assets/138105847/e687a3f7-6871-4756-855e-ed65435b7ac2">

**Comment by jlipps** [comment]
This is an issue report and not a discussion I think. Please fill out the issue template instead.
